lib/taperole/core/installer.rb

```diff
@@ -0,0 +1,87 @@
+require 'pathname'
+
+module Taperole
+  module Installer
+    include Taperole::Helpers::Files
+    include Thor::Actions
+
+    protected
+
+    def install_tape
+      add_tape_to_gitignore
+      mkdir tapefiles_dir
+      create_tape_files
+      create_roles_dir
+      create_inventory_file
+      create_ssh_keys_dir
+      handle_vagrantfile
+    end
+
+    def uninstall_tape
+      rm "#{tapefiles_dir}/omnibox.yml"
+      rm "#{tapefiles_dir}/deploy.yml"
+      rm "#{tapefiles_dir}/tape_vars.yml"
+      rm "#{tapefiles_dir}/rake.yml"
+      rm "#{tapefiles_dir}/roles"
+      rm "#{tapefiles_dir}/hosts"
+      rm "#{local_dir}/dev_keys"
+      rm "#{local_dir}/Vagrantfile"
+    end
+
+    private
+
+    def add_tape_to_gitignore
+      File.open('.gitignore', 'r+') { |f| f.puts '.tape' unless f.read =~ /^\.tape$/ }
+    end
+
+    def create_tape_files
+      if fe_app? && !rails_app?
+        logger.info '🔎  JS/HTML app detected'.red
+        copy_static_app_examples
+      elsif rails_app?
+        logger.info '🔎  Rails app detected'.red
+        copy_basic_examples
```

Comment:
just curious if there should be an `else` here in case the first two cases are false?  or else i'm guessing you are checking the app type elsewhere, too, so it's unnecessary here maybe.